portfolio/src/main/java/com/google/sps/servlets/StoreCard.java

```diff
@@ -52,8 +54,11 @@ public StoreCard(PlacesSearchResult store, GeoApiContext geoApiContext) {
 
         // If the store has their hours available, check if they are open.
         OpeningHours openingHours = store.openingHours;
-        if (openingHours != null) {
+        try {
             openNow = openingHours.openNow;
```

Comment:
I think the old way was better. You shouldn't need to throw an error especially if you are only printing the error.

Reply:
For some reason, the old way was giving a null pointer exception. Do you have any suggestions on how to handle it?